client/influxdb_test.go

```diff
@@ -517,8 +517,8 @@ func TestClient_Timeout(t *testing.T) {
 	_, err = c.Query(query)
 	if err == nil {
 		t.Fatalf("unexpected success.  expected timeout error")
-	} else if !strings.Contains(err.Error(), "use of closed network connection") {
-		t.Fatalf("unexpected error.  expected 'use of closed network connection' error, got %v", err)
+	} else if !strings.Contains(err.Error(), "request canceled") {
```

Comment:
That's interesting.  It appears to be a "better" error now.  I don't remember reading anything in the release notes on it, but it's probably in their changelog.  Seems fine. +1